src/Widgets/EligibilityModal/index.tsx

```diff
@@ -36,9 +36,11 @@ const EligibilityModal: FunctionComponent<Props> = ({
     status,
   }
 
+  const isDeffered = currentPlan?.deferred_days > 0 || currentPlan?.deferred_months > 0
+
   return (
     <Modal onClose={onClose} ariaHideApp={false} scrollable isOpen>
-      <ModalComponent {...modalProps}>
+      <ModalComponent {...modalProps} isDeffered={isDeffered}>
```

Comment:
I know this is out of scope, but the `{...modalProps}` is dead, can you please remove it (+ the `modalProps` constant)

Reply:
done